src/components/scenes/PluginViewScene.js

```diff
@@ -240,6 +241,38 @@ class PluginView extends React.Component<PluginProps, PluginState> {
       return
     }
     const parsedUrl = parse(navState.url, {}, true)
+    if (parsedUrl.query['edge-callback']) {
+      switch (parsedUrl.query['edge-callback']) {
```

Comment:
This is nearly all duplicate code from below. Can you consolidate this logic into common code. It looks like if someone used `x-callback-url` they would be missing some necessary logic like.

`openingSendConfirmation = true`

Reply:
moved the code down before removing this whole workflow